packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-pattern-details.js

```diff
@@ -101,6 +70,60 @@ export default function usePatternDetails( postType, postId ) {
 		} );
 	}
 
+	const templatePartAreaLabels = {
+		header: __( 'Header' ),
+		footer: __( 'Footer' ),
+		sidebar: __( 'Sidebar' ),
+		uncategorized: __( 'General' ),
+	};
+
+	if ( postType === 'wp_template_part' ) {
+		details.push( {
+			label: __( 'Area' ),
+			value: templatePartAreaLabels[ record.area ],
```

Comment:
We need to be able to handle custom registered template part areas here.

Reply:
This has been covered in https://github.com/WordPress/gutenberg/pull/52476/commits/d7a1b4efff37817093607cbaef0365b08ec61abf